src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -14,6 +14,13 @@ import FactorUtil from '../../../util/FactorUtil';
 
 const { getPasswordComplexityDescription, getPasswordComplexityDescriptionForHtmlList } = FactorUtil;
 
+const getButtonDataSeAttr = function (authenticator) {
+  const method = authenticator.value?.methodType ?
+    '-' + authenticator.value?.methodType : '';
+  const authenticatorType = authenticator.authenticatorType || '';
```

Comment:
is it possible that an authenticator does not have a type?

Reply:
Same answer as the other - Because `getAuthenticatorData` is used for both LEA factorTypes and IDX Authenticators, i'm returning `''` in case the object does not have authenticatorType. Otherwise the LEA tests will set `'undefined'` instead of `''`. I think this is something we could clean up when we remove LEA code. Similar code here - https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/AuthenticatorUtil.js#L19

Reply:
Got it. One suggestion though, should we rewrite this something like
```
if (authenticator.authenticatorType) {
  const method = authenticator.value?.methodType ? '-' + authenticator.value?.methodType : '';
  return authenticator.authenticatorType + method;
}
return ''; // or 'unknown-factor' if we want a string in the data-se attribute
```

Reply:
Will change to `''` for now 👍